.pylintrc

```diff
@@ -62,6 +62,7 @@ ignore-imports=yes
 #    too-many-statements
 # and fix those issues.
 
+disable=no-self-use,broad-except,too-many-statements,too-many-locals,too-many-branches,dangerous-default-value,not-context-manager,duplicate-code
```

Comment:
Could we try to not disable lint rules if possible? If there are particular ones that we want to change generally, let's have specific discussions about each one in turn. We're trying to move away from having to disable lint rules in the main codebase, but it seems better to just start this one off on the right foot.